### DISCUSSION TITLE
Support for Cargo Binstall

### DISCUSSION THREAD

**Discussion by erayerdin** [diesel-rs/diesel#3045] [Ideas]
[cargo-binstall](https://github.com/ryankurte/cargo-binstall) is a project letting prebuilt binaries to be downloaded instead of compiling the code locally like `cargo install` does. This would save a lot of time.

**Comment by weiznich** [comment]
That needs something like https://github.com/diesel-rs/diesel/issues/2379 first. That written: I'm not opposed to such a change, but at least I will likely not work on the implementation. So PR's are welcome here.
